openapi.yaml

```diff
@@ -3859,6 +3859,19 @@ paths:
                   $ref: '#/components/schemas/DatabaseMySQLRequest/properties/allow_list'
                 updates:
                   $ref: '#/components/schemas/DatabaseMySQL/properties/updates'
+                type:
+                  type: string
+                  description: |
+                    Request re-sizing of your cluster to a higher plan by specifying a larger Linode Type.
```

Comment:
The constraint is that the new Type can't have less disk space than the current Type. The term "higher plan" isn't entirely accurate here.

Reply:
Updated the statement to accommodate.